README.md

```diff
@@ -126,7 +139,85 @@ you: ~restart_app your_app_name_or_id
 crates-io-bot: @you: All dynos in your-app-name have been restarted.
 ```
 
-There will be more commands specific to managing the crates.io infrastructure very soon.
+**~update_app_config**
+
+You can update authorized application configuration variables through the ~update_app_config command.
+
+Authorized configuration variables are defined here:
+**heroku.rs**
+```rust
+// Config variables that can be updated through Discord
+const AUTHORIZED_CONFIG_VARS: &[&str] = &["FOO"];
+```
+
+Let's say you have an app called "testing-nell-bot". That app has a config variable with the key "FOO" and you want to update the value of that key to "bar". You would run this command:
+
+```
+you: ~update_app_config testing-nell-bot FOO bar
```

Comment:
Staging is a separate app, and it'd be awesome to manage it with this bot along with the production environment.

Reply:
For now, I'd prefer to require someone to specify the app name (especially since it is straightforward to get the app names through the ~get_apps command) since we will likely control both production and staging with this bot. My worry is if I default to production (or staging) people will accidentally alter the config vars for one environment when they mean the other.